.test-infra/jenkins/job_Release_Gradle_NightlySnapshot.groovy

```diff
@@ -36,13 +36,29 @@ job('beam_Release_Gradle_NightlySnapshot') {
       'dev@beam.apache.org')
 
 
-  // Allows triggering this build against pull requests.
+  // Allows triggering this publish command against pull requests.
```

Comment:
The publishing to the snapshot should not be triggerable by a trigger phrase, since it affects the state of anyone relying on the snapshot. Delete this trigger?

Reply:
Agree.

Reply:
I found we use this trigger phrase to trigger a snapshot build in release process: https://beam.apache.org/contribute/release-guide/#start-a-snapshot-build